src/shared/components/variant-alleles-sequences/VariantAllelesSequences.tsx

```diff
@@ -108,7 +108,9 @@ const prepareSequenceData = (props: Props) => {
     alleleSequences.unshift(referenceAllele.reference_sequence);
   }
 
-  const combinedString = alleleSequences.join('/');
+  const alleleSequencesCopy = alleleSequences.slice(0);
+  const combinedString =
+    alleleSequencesCopy.shift() + '  ' + alleleSequencesCopy.join(',');
```

Comment:
This line shows that you are trying to separate the reference allele from the alternative alleles. But we already had this separation above, and lost it on line 108. The fact that we need to keep the reference allele and alternative alleles separately suggests that it is worth rewriting the function, like so:

```
const prepareSequenceData = (props: Props) => {
  const { referenceAllele, alternativeAlleles } = getReferenceAndAltAlleles(
    props.alleles
  );

  // we expect reference allele to always exist in the data;
  // but default to an empty string to appease typescript
  const referenceAlleleSequence = referenceAllele?.reference_sequence ?? '';

  const alternativeAlleleSequences = alternativeAlleles.map(
    (allele) => allele.allele_sequence
  );

  const combinedString = `${referenceAlleleSequence} ${alternativeAlleleSequences.join(',')}`;

  return {
    referenceAlleleSequence,
    alternativeAlleleSequences,
    combinedString
  };
};
```

And then you can also update the `ExpandedView` component to treat the `referenceAlleleSequence` and the `alternativeAlleleSequences` differently. This makes the programmer's intention clearer than just treating the zeroth element of a string array differently.